docs/dev-guide.md

```diff
@@ -81,20 +82,15 @@ A shell script is provided to deploy the artifacts necessary to test the kraan-c
 To test kraan-controller
 
     scripts/run-controller.sh --help
-    USAGE: run-controller.sh [--debug] [--add-secret] [--ignore-test-errors] [--set-git-repo <repo url>] [--set-git-ref <branch name>]
```

Comment:
Are you intentionally removing the `--set-git-repo` and `--set-git-ref` options, or have you just not yet merged them into your local Git repo clone?

Reply:
Checked with you.  Sounds intentional and that's good enuf for me!  :-)